what4/test/ExprBuilderSMTLib2.hs

```diff
@@ -922,6 +922,62 @@ stringTest5 sym solver = withChecklist "string5" $
        _ -> fail "expected satisfable model"
 
 
+stringTest6 ::
```

Comment:
A comment here describing what property this is testing would be helpful, since the name `stringTest6` alone doesn't say much on its own.